src/vm/builtins/builtin_runner/bitwise.zig

```diff
@@ -211,7 +231,7 @@ pub const BitwiseBuiltinRunner = struct {
             else => return BitwiseError.InvalidBitwiseIndex,
         };
 
-        return MaybeRelocatable{ .felt = Felt252.fromInteger(res) };
+        return .{ .felt = Felt252.fromInteger(res) };
```

Comment:
Maybe here we can directly use: 

```zig
MaybeRelocatable.fromU256( switch (index) {
            2 => x & y, // and
            3 => x ^ y, // xor
            4 => x | y, // or
            else => return BitwiseError.InvalidBitwiseIndex,
        }
)
```

method no?

Reply:
cool idea!